apps/vs-code-designer/src/app/commands/deploy/deploy.ts

```diff
@@ -198,7 +199,11 @@ async function deploy(
 
     try {
       if (isHybridLogicApp) {
-        await deployHybridLogicApp(context, node);
+        if (canUseZipDeployForHybrid(node)) {
+          await deployHybridLogicAppV2(context, node, effectiveDeployFsPath);
```

Comment:
Can we have a better description rather than just adding V2? What does the V2 does? whats different than normal one?